NOTICE

```diff
@@ -0,0 +1,6 @@
+Druidry - a Java Druid Client
+Copyright 2018-present ZAPR Media Labs
+
+This product includes software developed at
+ZAPR Media Labs (https://www.zapr.in/)
+
```

Comment:
Have we used any licensed libraries after modifying them ? 
If that derivative work is under Apache License, I think we might need to mention them in NOTICE (as per 4d of T&Cs).

Reply:
We are using compiled version of libraries as it is.